specification-components.md

```diff
@@ -0,0 +1,107 @@
+# Reusing Descriptions
+
+The examples given so far have started to grow large because of the verbose nature of OpenAPI documents. This page introduces a mechanism to remove redundancy from the document by reusing portions of it.
```

Comment:
Dunno if we want to talk about verbosity here, its a common complaint from people who are not very experienced with it, and think OpenAPI == 435346346 lines of YAML in a single file.

Reply:
How does this sound?
> As is often the case, the example built through the previous pages has grown too large to be easily manageable. This page introduces a mechanism to remove redundancy from the document by reusing portions of it.

Reply:
@philsturgeon , @segfaultxavi  - sounds good to me...

Reply:
done!